File: dry_cleaning_system_HmOS/entry/src/main/ets/models/RechargeRecord.ts
/**
 * 充值记录数据模型
 */
export interface RechargeRecord {
  id: number;
  customerId: number;
  rechargeAmount: number;
  giftAmount: number;
  createTime: string;
}

/**
 * 充值请求
 */
export interface RechargeRequest {
  customerId: number;
  amount: number;
}

File: dry_cleaning_system_Android/app/src/main/java/com/chaser/drycleaningsystem/data/entity/RechargeRecord.kt
package com.chaser.drycleaningsystem.data.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * 充值记录实体类
 */
@Entity(tableName = "recharge_record")
data class RechargeRecord(
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
    @ColumnInfo(name = "customer_id") val customerId: Long,
    @ColumnInfo(name = "recharge_amount") val rechargeAmount: Double,
    @ColumnInfo(name = "gift_amount") val giftAmount: Double,
    @ColumnInfo(name = "create_time") val createTime: String = System.currentTimeMillis().toString()
)
